### ISSUE TITLE
python 3.4 detection (Trac #732)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#726]
14:04:36 - JoaoPedr1: in cmake tools/python.cmake when looking for the python version the ERROR_VARIABLE is read. When I updated to python 3.4 with archlinux it stopped being able to get the version, changing ERROR_VARIABLE to OUTPUT_VARIABLE solved the problem

Sounds like a change from printing the version to stderr, to printing to stdout.

Migrated from https://code.icecube.wisc.edu/ticket/732
```json
{
    "status": "closed", 
    "changetime": "2019-01-11T23:10:39", 
    "description": "14:04:36 - JoaoPedr1: in cmake tools/python.cmake when looking for the python version the ERROR_VARIABLE is read. When I updated to python 3.4 with archlinux it stopped being able to get the version, changing ERROR_VARIABLE to OUTPUT_VARIABLE solved the problem\n\nSounds like a change from printing the version to stderr, to printing to stdout.", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1547248239669835", 
    "component": "cmake", 
    "summary": "python 3.4 detection", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2014-06-13T19:31:21", 
    "milestone": "", 
    "owner": "nega", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20140613T19:33:18: 
* david.schultz changed owner from "" to "nega"
* david.schultz changed status from "new" to "assigned"

**Comment by icecube-trac** [comment]
Trac update at 20140613T19:39:30: nega changed status from "assigned" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20141204T20:08:33: 
* nega changed resolution from "" to "fixed"
* nega commented:

> In changeset 2012:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2012"
> fix python3 detection
> 
> `python2 -V` goes to sdterr, `python3 -V` goes to stdout.
> 
> closes #726
> ```


* nega changed status from "accepted" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190111T23:10:39: nwhitehorn commented:

> In changeset 169906:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="169906"
> fix python3 detection
> 
> `python2 -V` goes to sdterr, `python3 -V` goes to stdout.
> 
> closes #726
> ```
